connectors/src/connectors/intercom/index.ts

```diff
@@ -166,6 +199,12 @@ export async function cleanupIntercomConnector(
 
   return sequelize_conn.transaction(async (transaction) => {
     await Promise.all([
```

Comment:
I know it's pre-existing code, but IMO we don't want `nangoDeleteConnection` inside the transaction

Reply:
I added a commit, thanks!